feat(datamodel): Write data model to InfluxDB for Grafana visualization

JBO5m4r7135 opened the pull request.

JBO5m4r7135 committed:
feat(datamodel): Write data model to InfluxDB for Grafana visualization

MathieuSabarthes commented on config.yaml:
```diff
@@ -29,12 +29,20 @@ Log:
+    Bucket : "demo-bucket"
+    Org : "demo-org"
+    Token : "Gj_nwgoqXu4XggIzaiUWFHALKnS5JskdTLzGlSYeJMNkjCD-pyR6Yc5Hvl8NWj5Qo5C80mLvuJAS1IuqOcq4GQ=="
+    IpAddress : "http://127.0.0.1"
```
Retirer http

MathieuSabarthes commented on ems/data/csv.go:
```diff
@@ -205,411 +164,41 @@ func NewEnergyGuardData(dataModelConfig config.DataModelConfig) *EnergyGuardData
+	h.logger.Info(message)
+}
+
+func writeTitle(fileName string) {
```
```suggestion
func writeHeaders(fileName string) {
```

MathieuSabarthes commented on ems/data/csv.go:
```diff
@@ -205,411 +164,41 @@ func NewEnergyGuardData(dataModelConfig config.DataModelConfig) *EnergyGuardData
-	writer.Comma = CSV_COLUMN_SEPARATOR
-	writer.Write(CSV_COLUMNS)
+	writer.Comma = column_separator
+	writer.Write(titles)
```
```suggestion
	writer.Write(headers)
```

MathieuSabarthes commented on ems/data/synchronizer.go:
```diff
@@ -0,0 +1,405 @@
+	return hasChanged
+}
+
+func (s *DataSynchronizer) GeMeter() (meter linkymeter.MeterData) {
```
```suggestion
func (s *DataSynchronizer) GetMeter() (meter linkymeter.MeterData) {
```

MathieuSabarthes commented on ems/data/synchronizer.go:
```diff
@@ -0,0 +1,405 @@
+	return hasChanged
+}
+
+func (s *DataSynchronizer) GeMeterMinAvailableCurrent() (minAvailableCurrent float64) {
```
```suggestion
func (s *DataSynchronizer) GetMeterMinAvailableCurrent() (minAvailableCurrent float64) {
```

MathieuSabarthes commented on config.yaml:
```diff
@@ -29,12 +29,20 @@ Log:
+    Token : "Gj_nwgoqXu4XggIzaiUWFHALKnS5JskdTLzGlSYeJMNkjCD-pyR6Yc5Hvl8NWj5Qo5C80mLvuJAS1IuqOcq4GQ=="
+    IpAddress : "http://127.0.0.1"
+    TcpPort : 8086
 # Tele indformation client (TIC) configuration
```
```suggestion
# Tele information client (TIC) configuration
```

MathieuSabarthes commented on ems/data/model.go:
```diff
@@ -0,0 +1,69 @@
+	Diagnosis          DiagnosisData          `json:"Diagnosis"`
+}
+
+func IsOverloadProtectionEqual(overloadProtection OverloadProtectionData, other OverloadProtectionData) bool {
```
Pourquoi tu ne fais cette vérification que pour OverloadProtection et pas pour les autres ?

JBO5m4r7135 committed:
chore: Update influxDb config removing 'http' from IpAddress and Token value using environment variable

Replace CSV title with 'headers'.
Replace IsOverloadProtectionEqual function with cmp.Equal function
Rename GeMeter function as GetMeter
Rename GeMeterMinAvailableCurrent as GetMeterMinAvailableCurrent

JBO5m4r7135 committed:
chore: Rename indformation as information in config file

JBO5m4r7135 committed:
chore: Rename GeMeter, GeMeterMinAvailableCurrent, GeOverloadProtection functions
